tests/core/test_callable_utils.py

```diff
@@ -155,7 +160,7 @@ def my_multiline_arg_function(a: int, b: str) -> str:
     assert sql_body.strip() == expected_sql.strip()
 
 
-def test_function_with_extra_docstring_params_ignored():
+def test_function_with_extra_docstring_params_ignored(recwarn):
```

Comment:
Woo this is cool!

Reply:
I've always liked the standard library recorder for this stuff (makes the tests shorter)